src/pygeosx/CMakeLists.txt

```diff
@@ -21,11 +21,11 @@ add_custom_command( TARGET pygeosx
 
 add_custom_command( TARGET pygeosx
                     POST_BUILD
-                    COMMAND ln -sf ${CMAKE_BINARY_DIR}/lib/pygeosx.so ${CMAKE_BINARY_DIR}/lib/PYGEOSX/lib/python3.8/site-packages/ )
+                    COMMAND ln -sf ${CMAKE_BINARY_DIR}/lib/pygeosx.so ${CMAKE_BINARY_DIR}/lib/PYGEOSX/lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages/ )
```

Comment:
@cssherman Does that sound good to you?
I tend to be OK but I don't really know what's going on there.

Reply:
Looks great - I had to do the same in a test branch myself recently